.changes/next-release/Feature-34fb084d-90b6-4644-9b6b-4459ac636994.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "IAM Identity Center (SSO): When signing in, verify the device code matches instead of pasting it"
```

nit: it's also builder id so could avoid some inaccuracy here

```suggestion
	"description": "Authentication: When signing in to AWS Builder Id or IAM Identity Center (SSO), verify the device code matches instead of copy-pasting it"
```